packages/@okta/vuepress-site/docs/guides/common-hook-set-up-steps/main/index.md

```diff
@@ -63,7 +63,7 @@ To add the `npm` packages:
 
 ## Add authentication method
 
-Okta inline hooks can use header authentication as well as OAuth 2.0 authentication to secure the calls from Okta to your external service. The inline hook Glitch projects use basic authentication. See the following sections to implement basic authentication or to use the OAuth 2.0 client secret or the private key method.
+Okta inline hooks can use header authentication as well as OAuth 2.0 authentication to secure the calls from Okta to your external service. The inline hook Glitch projects use Basic Authentication. See the following sections to implement Basic Authentication or to use the OAuth 2.0 client secret or the private key method.
```

Comment:
did I agree to this? If I did, I was wrong. Why are we capitalizing this? It doesn't need to be.

Reply:
@susanharper-okta Because it's referring to HTTP Basic Authentication which is a proper noun and so is capitalized.